service/src/main/java/bio/terra/drshub/config/DrsProviderInterface.java

```diff
@@ -122,14 +119,6 @@ && getAccessMethodConfigTypes().contains(AccessMethodConfigTypeEnum.gs)
         && Fields.overlap(requestedFields, Fields.BOND_SA_FIELDS);
   }
 
-  default boolean shouldFetchPassports(AccessMethod.TypeEnum accessMethodType) {
```

Comment:
We don't need this anymore because Drs Providers now tell us if they need a passport!